prereise/gather/demanddata/nrel_efs/map_states.py

```diff
@@ -52,6 +101,27 @@ def decompose_demand_profile_by_state_to_loadzone(df, save=None):
     # Convert from local hours to UTC time
     df_lz = shift_local_time_by_loadzone_to_utc(df_lz)
 
+    # Create mapping from state abbreviations to loadzone IDs
+    abv2id = {}
+    for k, v in id2abv.items():
+        abv2id.setdefault(v, []).append(k)
```

Comment:
Or we could define `abv2id` as a `defaultdict(list)` here.

Reply:
Yeah, I can change it it use `defaultdict` instead. Is there a reason this is better? Also, this mapping doesn't appear to be needed elsewhere, which is why I needed to create it here. That being said, should this be added to all of the other mappings in `powersimdata\network\usa_tamu\constants\zones.py`?

Reply:
Good idea, let's add it in.

Reply:
Okay, I'll add it in there instead.

Reply:
Done.